.mergify.yml

```
@@ -0,0 +1,35 @@
+pull_request_rules:
+  - name: Label the approved PRs
+    conditions:
+      - "#approved-reviews-by>=1"
```

```suggestion
      - "#approved-reviews-by>=1"
      - "#changes-requested-reviews-by=0"
      - "#review-threads-unresolved=0"
      - "check-success=Test added-OFEP proposals for status"
```

This matches the branch protection rules.